.github/workflows/patch_build.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       - name: Run Doc Tests
         run: |
           docker run -d --name toolkit_test -e POSTGRES_HOST_AUTH_METHOD=trust -p 5432:5432 timescaledev/timescale-analytics:nightly
-          cargo run --manifest-path ./tools/sql-doctester/Cargo.toml -- -h localhost -s "CREATE EXTENSION timescaledb_toolkit; SET SESSION TIMEZONE TO 'UTC'" -p 5432 docs
+          cargo run --manifest-path ./tools/sql-doctester/Cargo.toml -- -h localhost -p 5432 docs
```

Comment:
This CI job has bitrotted and always fails to run (it's only run when certain files are modified). The patch Docker image should probably be completely removed in a future PR.

Reply:
motion seconded

Reply:
I asked @JLockerman if I could get rid of it the first time I tripped over it (PR #442).  At the time, he thought it was still useful, though I can't find any written record of our conversation about it.

I think it's pretty clear we don't need it now...

Reply:
Yeah this thing only ever had a very specific use–getting a version of the current TimescaleDB docker image with the latest dev version of Toolkit installed–and was only ever useful as an internal debugging tool. It was always rather brittle, and if we're not finding it useful anymore we might as well delete it.